rate_counter/rate/tasks.py

```diff
@@ -1,11 +1,19 @@
 # -*- encoding: utf-8 -*-
-from __future__ import absolute_import
+
+from rate import *
 from celery.task import periodic_task
-from .models import Institute, StudentGroup, Subject
```

Comment:
Ok, tested - this brake task execution, Please fix. Actually that's not a surprise, as  
`Institute, StudentGroup` and maybe smth else were removed from import.